src/Language/Rust.ts

```diff
@@ -332,26 +332,23 @@ class RustRenderer extends ConvenienceRenderer {
     };
 
     protected emitUsageExample(): void {
+        const topLevelName = this.topLevels.keys().next().value;

```

Comment:
If you want to get the first top level name, use `defined(this.topLevels.keySeq().first())`